### ISSUE TITLE
[research] AERA-v26.9 CUDA timeline dispatch-gap attribution successor after #701 ref-order incident

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#702]
## Purpose

Fresh implementation-governance successor for the diagnostic question preregistered in #701, after the #701 pre-freeze ref-order incident recorded in comment `5571988281`.

This issue does **not** change the scientific question, parent systems evidence, candidate architecture, checkpoint, routing, thresholds, or authority. It exists solely to restore the strict order: issue assignment → detached issue-specific blobs → detached tree/commit → exact freeze comment → only then implementation branch/ref move → one immutable PR CPU attempt.

## Source and parent evidence

Bind source main/tree at issue creation:
- `SOURCE_MAIN=6fad37d0e4c611fcaff480899b25dc190e9e9127`
- `SOURCE_TREE=87a22d981caf77263574503749757ba77290cc9a`

Canonical scientific parent remains #700/#697:
- `PARENT_TRIGGER=700`
- `PARENT_EVIDENCE_COMMENT=5571118103`
- `PARENT_RUN=34125640230`
- `PARENT_JOB=101753558368`
- `PARENT_ATTEMPT=1`
- source decision `FAIL_FROZEN_E2E_SYSTEMS_GATE`
- common lower-level target `dispatch_or_unattributed_gap`
- batch 8 normalized impact `9.2633691290416 ms`, parent profiler distortion `2.6684698738709085x`
- batch 64 normalized impact `12.370303117411762 ms`, parent profiler distortion `1.6960100325791625x`
- `optimization_authorized=false`, `systems_pass_earned=false`, all higher-stage flags false.

Governance predecessor:
- issue #701
- incident comment `5571988281`
- unintended inert branch `__noop_probe_do_not_create` points only to source main and receives zero authority; it must never be used as this issue's branch, PR head, trigger, source, or evidence.

## Diagnostic boundary

Diagnostic-only. Execute the exact frozen v26.9 candidate with `route_mode="hard_sparse"`, `hard=True`, `update_memory=True`, CUDA BF16 autocast, seed8471 checkpoint, batches `(8,64)`, token seed rule `138471 + 10000 + batch_size`.

Use low-intrusion PyTorch/Kineto CUDA timeline measurement plus separately measured unprofiled CUDA-event baselines. Persist chronological device activity, unioned active time, inter-activity idle gaps, exposed CUDA runtime launch/copy/wait data, repeated small device activity families, large kernel families, profiler distortion, and direct frozen-model route/executed-fraction evidence. Explicitly mark unavailable fields rather than infer them.

For kernel-vs-ATen attribution, classify only where trace correlation is explicitly available; incomplete correlation must disable that mechanism classification rather than guess.

## Decision rule

Materiality remains exactly:
- `MATERIAL_MS_MIN=1.0`
- `MATERIAL_SHARE_MIN=0.15`

Exactly one `next_target` may be named only if it is the largest common material candidate at both frozen batches, from:
- `launch_or_idle_reduction`
- `repeated_small_kernel_fusion`
- `explicit_device_kernel_optimization`

Otherwise `next_target=None`. Unavailable/unattributed residual cannot authorize optimization.

Even a classification authorizes no optimization automatically. A separately preregistered optimization successor remains required, and any optimization microbenchmark still cannot earn systems PASS without a fresh full frozen Transformer-relative E2E systems gate.

## Implementation governance

Use fresh issue-specific namespaces only. Before **any** implementation branch/ref move:
1. create exactly three additive #702-specific candidate blobs as detached Git objects;
2. build the exact detached tree from source tree `87a22d981caf77263574503749757ba77290cc9a`;
3. build the exact detached commit with source main as sole parent;
4. post one exact pre-implementation freeze comment containing source main/tree, branch, preauth/L4 prefixes, result paths, parent evidence, thresholds, all frozen dependency blobs, three new blobs, frozen tree, frozen commit, and the #701 incident exclusion;
5. only then create/move the #702 research branch to that exact frozen commit.

Prefer exactly three additive files: one Modal launcher, one owner-only issues-opened workflow, one CPU/static contract test. Package every root-level remote dependency explicitly.

Require one immutable PR head and one original full-repository CPU CI attempt. Never rerun/retry/redispatch a consumed attempt. Merge only after fresh live audit and explicit authorization, and merge only the exact frozen head if CPU-green.

After merge: fresh exact-main/tree/blob/result-absence audit, exactly one fresh zero-GPU preauthorization trigger under the frozen #702 preauth prefix, separate exact-main L4 authorization, then exactly one fresh #702 L4 trigger.

No GPU execution, optimization, systems PASS, architecture freeze, S2, fresh scientific seed, independent replication credit, 100M scaling, or breakthrough claim is authorized by this issue itself.

**Comment by vinceackermann2-sys** [comment]
## #702 pre-implementation CUDA timeline attribution freeze

SOURCE_MAIN=6fad37d0e4c611fcaff480899b25dc190e9e9127
SOURCE_TREE=87a22d981caf77263574503749757ba77290cc9a
GOVERNANCE_PREDECESSOR_ISSUE=701
GOVERNANCE_INCIDENT_COMMENT=5571988281
EXCLUDED_INERT_REF=__noop_probe_do_not_create
BRANCH=research/aera-v26-9-issue702-cuda-timeline-dispatch-gap-attribution
PREAUTH_PREFIX=[aera-v26-9-issue702-cuda-timeline-preauth]
L4_PREFIX=[aera-v26-9-issue702-cuda-timeline-l4]
RESULT_PATH=/vol/aera-v26/issue702-cuda-timeline-dispatch-gap-attribution/result.json
PARENT_RESULT_PATH=/vol/aera-v26/issue697-micro-op-attribution/result.json
PARENT_TRIGGER=700
PARENT_EVIDENCE_COMMENT=5571118103
PARENT_RUN=34125640230
PARENT_JOB=101753558368
PARENT_ATTEMPT=1
SOURCE_DECISION=FAIL_FROZEN_E2E_SYSTEMS_GATE
PARENT_NEXT_TARGET=dispatch_or_unattributed_gap
BATCH8_PARENT_GAP_MS=9.2633691290416
BATCH8_PARENT_PROFILER_DISTORTION=2.6684698738709085
BATCH64_PARENT_GAP_MS=12.370303117411762
BATCH64_PARENT_PROFILER_DISTORTION=1.6960100325791625
CHECKPOINT_AERA_SHA256=f8aa92421801e8f190247e420632be5f0c20bc5ea8bf6bdeefe06686b3a31b30
CHECKPOINT_TRANSFORMER_SHA256=cdd5cab4439a709468d6607d45d82081b33e876b2e40d91d4a38ba139b219dd7
ISSUE697_LAUNCHER_BLOB=c82affc01d1ecb096371e8865dbd7b18e739f5e1
ISSUE687_LAUNCHER_BLOB=b9950c032686a496c6d944c30c441428499cb367
ISSUE665_LAUNCHER_BLOB=72f27391ff2f0a7bff8d4532f307ddc4869cf494
SCIENTIFIC_ADAPTER_BLOB=512572340cc09e2e7ad6729712258c12cb377ef2
BASE_SYSTEMS_BLOB=c9731cae7e386f09b2a190b045532591c4fa00be
RUNTIME_INTERFACE_BLOB=268644ac4edee15a4cc4e29d3fed7f61eeb3caa7
STAGE_V25_1_BLOB=1c3456d8040455b4cd1194db4c8586f77d0f3e43
COMPACT_V25_1_BLOB=4e336b6e1a6238dac782fa320751d68281493ee1
TOKENWISE_V19_BLOB=98008bceb8c68af3bc346e5dfcc7a8218875661e
BASE_CORE_BLOB=ffe0341829fb905e40ef9ae3544f797471f1aa9c
NOHOST_V25_1_BLOB=237e5615cf32f644e8675808a6b3e9adaf04fb23
V26_9_BACKEND_BLOB=b81cc209f5d95abbe1fb8bd620c78e87c067bc19
TRIAGE_BLOB=e5ce8cdda0777dce97816e2640f4492803a6b191
LAUNCHER_BLOB=bf38814a976d929119d17b9e907f1b89461a14a2
WORKFLOW_BLOB=89ce600b3b0111379cf856404e3c7c4ec2d3a0e1
CPU_TEST_BLOB=90d39753c55bf3e5f4330339df15c79f2ae31ac5
FROZEN_TREE=27be25ba578aa530c1e6d832db1a38be4c8bc22a
FROZEN_COMMIT=77925aa550b7078cc799145cc3fcd8ef78195b77
BATCHES=8,64
TOKEN_SEED_RULE=138471+10000+batch_size
ROUTE_MODE=hard_sparse
HARD=true
UPDATE_MEMORY=true
CUDA_BF16_AUTOCAST=true
BASELINE_CALLS=3
PROFILE_CALLS=1
SMALL_KERNEL_MAX_US=50.0
REPEATED_MIN_COUNT=2
MATERIAL_SHARE_MIN=0.15
MATERIAL_MS_MIN=1.0
MAX_GPU_SECONDS=300

Frozen commit `77925aa550b7078cc799145cc3fcd8ef78195b77` has sole parent `6fad37d0e4c611fcaff480899b25dc190e9e9127`, exact tree `27be25ba578aa530c1e6d832db1a38be4c8bc22a`, and exactly three additive #702 files / 775 additions / 0 deletions. No existing scientific/runtime/model/checkpoint/evaluator file is modified.

The #701 candidate blobs and `__noop_probe_do_not_create` ref are explicitly excluded and receive zero authority. Only the three #702 blobs/tree/commit above may become the #702 implementation PR head.

Diagnostic rules are frozen: one NVIDIA L4 only after successful merged zero-GPU preauthorization plus separate exact-main L4 authorization; no training, optimizer, backward, corpus, scientific-seed consumption, comparative systems gate rerun, reference execution, Transformer execution, or checkpoint write. Kernel-based mechanism classification is disabled if observed kernel↔ATen correlation is incomplete; unavailable/unattributed residual cannot authorize a mechanism.

This freeze itself authorizes only moving the named #702 research branch to the exact frozen commit and opening one immutable CPU-validation PR. It authorizes no Modal execution, GPU execution, optimization, systems PASS, architecture freeze, S2, fresh scientific seed, independent replication, 100M scaling, or breakthrough claim.

**Comment by vinceackermann2-sys** [comment]
## #702 CPU-green exact-head merge authorization

Authorize merge of PR #703 only.

- Source main before merge: `6fad37d0e4c611fcaff480899b25dc190e9e9127`, tree `87a22d981caf77263574503749757ba77290cc9a`.
- Exact authorized PR head: `77925aa550b7078cc799145cc3fcd8ef78195b77`.
- Exact authorized head tree: `27be25ba578aa530c1e6d832db1a38be4c8bc22a`.
- Exact diff: 3 additive #702 files / 775 additions / 0 deletions.
- Freeze comment: `5572124225`.
- Sole original full-repository CPU CI: run `34133459841`, job `101778837313`, attempt `1`.
- CI result: `1107 passed, 15 warnings in 138.08s`.
- Merge method: `merge`.
- Expected-head guard required: `77925aa550b7078cc799145cc3fcd8ef78195b77`.
- #701 incident comment `5571988281` and inert ref `__noop_probe_do_not_create` remain excluded from authority.

Fresh live audit immediately before this authorization confirmed `main` is unchanged, PR #703 is open/mergeable at the exact frozen head, the sole CI attempt is successful attempt 1, and no newer scientific/successor issue has advanced this line.

This authorization permits only the exact merge above. The merge authorizes no Modal execution, GPU execution, diagnostic result, optimization, systems PASS, architecture freeze, S2, fresh scientific seed, independent replication, 100M scaling, or breakthrough claim. After merge, a fresh exact-main audit and a separately consumed zero-GPU #702 preauthorization trigger are required before any L4 authorization.

**Comment by vinceackermann2-sys** [comment]
## #702 sole L4 CUDA timeline attribution authorization

Authorize main: `4883abe053347379812901c5c2057da70c11e3e7`
Authorize tree: `27be25ba578aa530c1e6d832db1a38be4c8bc22a`

Successful prerequisite zero-GPU preauthorization is exactly:
- trigger issue `#704`
- evidence comment `5572266697`
- workflow run `34134251152`
- job `101781353804`
- attempt `1`
- `result_absent=true`
- `gpu_used=false`
- `model_constructed=false`
- `new_measurement_performed=false`

Authorize exactly one fresh trigger under `[aera-v26-9-issue702-cuda-timeline-l4]` bound to the exact merged main above. Execute only frozen launcher blob `bf38814a976d929119d17b9e907f1b89461a14a2` on one NVIDIA L4, with result path `/vol/aera-v26/issue702-cuda-timeline-dispatch-gap-attribution/result.json`, exact checkpoint/source blobs, batches `(8,64)`, token seed rule `138471 + 10000 + batch_size`, hard-sparse routing, `hard=True`, `update_memory=True`, CUDA BF16 autocast, `BASELINE_CALLS=3`, `PROFILE_CALLS=1`, `SMALL_KERNEL_MAX_US=50.0`, `REPEATED_MIN_COUNT=2`, `MATERIAL_SHARE_MIN=0.15`, and `MATERIAL_MS_MIN=1.0`.

Purpose only: resolve the frozen `dispatch_or_unattributed_gap` into observed device activity / idle gaps / runtime launch-copy-wait activity / repeated small kernels / large kernels and, only if the preregistered common-material rule is met at both batches, name one diagnostic `next_target`.

This authorization does not authorize training, optimizer/backward, checkpoint mutation, comparative systems rerun, optimization implementation, systems PASS, architecture freeze, S2, fresh scientific seed, independent replication credit, 100M scaling, or breakthrough claim. The forthcoming L4 trigger and its Actions attempt are consumed regardless of outcome; never rerun/retry/redispatch.
